allow formatted byte size when humanizing ByteRate

ronnieoverby opened the pull request:
I have needed this on more than one occasion and have had to resort to not using ByteRate because I couldn't format the byte size part of the string.

mexx commented:
@ronnieoverby Thank you for your contribution.
Please take a look at the [contribution guideline](https://github.com/Humanizr/Humanizer/blob/dev/CONTRIBUTING.md#contribution-guideline).

The build is failing, could you please fix it.
Please update the `readme.md` with the extended functionality.

ronnieoverby commented:
I've updated the readme. I'm not sure how I can fix the failing build, though. It seems to be failing when verifying the public api didn't change. I added an overload and didn't change the public api.

Can you help me out?

mexx commented:
Sure, as you figured out, it fails on public api verification. The public api consist of every publicly available type, method signature and property.
The added overload is part of the public api. You have to update the [approve_public_api.approved.txt](https://github.com/Humanizr/Humanizer/blob/dev/src/Humanizer.Tests/ApiApprover/PublicApiApprovalTest.approve_public_api.approved.txt)

ronnieoverby commented:
Thanks, I didn't realize I had to update that file. For some reason, I was
thinking that the api approver was making sure I didn't remove/change
existing signatures only. I'll give it another shot.

On Fri, Jan 29, 2016 at 3:47 PM, Max Malook <notifications@github.com>
wrote:

> Sure, as you figured out, it fails on public api verification. The public
> api consist of every publicly available type, method signature and property.
> The added overload is part of the public api. You have to update the
> approve_public_api.approved.txt
> <https://github.com/Humanizr/Humanizer/blob/dev/src/Humanizer.Tests/ApiApprover/PublicApiApprovalTest.approve_public_api.approved.txt>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Humanizr/Humanizer/pull/517#issuecomment-176963006>.
>

ronnieoverby commented:
woohoo

onovotny commented:
Thanks!
